src/crimson/osd/osd_operations/snaptrim_event.h

```diff
@@ -153,12 +138,6 @@ class SnapTrimObjSubEvent : public PhasedOperationT<SnapTrimObjSubEvent> {
   remove_or_update_iertr::future<ceph::os::Transaction>
   remove_or_update(ObjectContextRef obc, ObjectContextRef head_obc);
 
-  // we don't need to synchronize with other instances started by
-  // SnapTrimEvent; it's here for the sake of op tracking.
```

Comment:
Should we reference a follow up tracker here to re-implement stages op tracking / event flags?

Reply:
Added a comment.